L1Trigger/Phase2L1ParticleFlow/plugins/L1SeedConePFJetProducer.cc

```diff
@@ -32,31 +33,37 @@ class L1SeedConePFJetProducer : public edm::global::EDProducer<> {
   unsigned nJets;
   bool HW;
   bool debug;
+  bool doCorrections;
```

Comment:
This could be `const` (also the others above, technically).

Reply:
this has been changed